pmm-app/src/pmm-update/hooks/usePerformUpdate.ts

```diff
@@ -8,23 +8,18 @@ export const usePerformUpdate = (): UpdateStatus => {
   const [updateFailed, setUpdateFailed] = useState(false);
   const [errorMessage, setErrorMessage] = useState('');
   const [output, setOutput] = useState('');
-  const [isUpdated, setIsUpdated] = useState(false);
+  const [updateFinished, setUpdateFinished] = useState(false);
   const [timeoutId, setTimeoutId] = useState<number>();
 
-  const [authToken, logOffset, initializationFailed, launchUpdate] = useInitializeUpdate();
+  const [authToken, initialLogOffset, initializationFailed, launchUpdate] = useInitializeUpdate();
 
   useEffect(() => {
-    if (!authToken || typeof logOffset === 'undefined') {
+    if (!authToken || initialLogOffset === undefined) {
       return;
     }
 
-    const updateStatus = async (logOffset: number, errorsCount = 0, isUpdated = false) => {
-      if (isUpdated) {
-        setIsUpdated(isUpdated);
-
-        return;
-      }
-
+    const updateStatus = async (logOffset: number, errorsCount = 0) => {
```

Comment:
Major readability improvement: no need to pass these parameters to `updateStatus` on every call -> it's enough to declare the two var in the outer scope.